app/controllers/achievements_controller.rb

```diff
@@ -14,12 +14,7 @@ def create
 
       @achievement.transition_to(:complete, metadata)
 
-      if @achievement.programme
-        case @achievement.programme.slug
-        when 'i-belong', 'primary-certificate', 'secondary-certificate'
-          CertificatePendingTransitionJob.perform_now(@achievement.programme, current_user.id, source: 'AchievementsController.create')
-        end
-      end
+      CertificatePendingTransitionJob.perform_now(current_user, { source: 'AchievementsController.create' })
```

Comment:
It's common practice to omit braces when passing arguments like this. The code being replaced here omits them so my assumption would be it's standard within this project to omit them too.

Reply:
`CertificatePendingTransitionJob.perform_now(current_user, source: 'AchievementsController.create')`

Reply:
I'm specifically passing in the hash as the second positional argument. The def perform(...) has kwargs meaning that if I passed just `source: '...'` it would throw an error due to a non-existent kwarg

Reply:
Well I stand corrected